packages/lib/src/toolbar/controls/ExportMenu.tsx

```diff
@@ -13,8 +13,8 @@ import { useId, useRef, useState } from 'react';
 import { FiDownload } from 'react-icons/fi';
 import { MdArrowDropDown } from 'react-icons/md';
 
+import toolbarStyles from '../Toolbar.module.css';
 import { useFloatingDismiss } from './hooks';
-import styles from './Selector/Selector.module.css';
```

Comment:
The styles shared between `Selector` and `ExportMenu` now live in `Toolbar.module.css`.